### PR TITLE
TBD: Persistent cache for packaging frontends

### PR DESCRIPTION
**Pull request by gaborbernat**
Signed-off-by: Bernát Gábor <gaborjbernat@gmail.com>

### CODE REVIEW

File: pep-9999.rst [hunk 1]
```diff
@@ -0,0 +1,65 @@
+PEP: 9999
+Title: Persistent cache for packaging frontends
+Author: Bernát Gábor <gaborjbernat@gmail.com>
+Sponsor: TBD
+Discussions-To: https://discuss.python.org/t/proposal-adding-a-persistent-cache-directory-to-pep-517-hooks/2303/57
```
Comment (uranusjr) [inline pep-9999.rst:5]: Would it be better to start a new thread instead? This thread is already quite long and can be difficult to follow for newcomers.

File: pep-9999.rst [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+The backend should use the ``cache_directory`` to save intermediate data.
+
+Build frontend requirements
+---------------------------
+The build frontend must persist the content within the cache directory, such as subsequent calls can recall data saved in prior calls. The build frontend must not share this folder across different projects. The folder must be empty during the first backend call.
+
+References
+==========
+
+.. _`PEP-517`: https://www.python.org/dev/peps/pep-0517/
```
Comment (uranusjr) [inline pep-9999.rst:50]: PEPs are automatically linked, so this is not needed, and all references can simply use `` :pep:`517` ``.

**Comment by brettcannon** [comment]
Please word wrap the text. See https://www.python.org/dev/peps/pep-0012/#general for guidance.

File: pep-9999.rst [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+Type: Standards Track
+Content-Type: text/x-rst
+Created: 02-Jun-2021
+Post-History: 
+
+
+Abstract
+========
+
+This document describes extensions to the build backend and frontend communication (as introduced by `PEP-517`_) to allow frontends to manage build backend
```
Comment (brettcannon) [inline pep-9999.rst:16]: PEPs are automatically linked.
```suggestion
This document describes extensions to the build backend and frontend communication (as introduced by PEP 517) to allow frontends to manage build backend
```

**Review by brettcannon — CHANGES_REQUESTED** [review]
For final merging, the PEP references should be plain text for automatic linking and the text needs to be reflowed as appropriate.

**Comment by gaborbernat** [comment]
I'll close this for now and reopen it later when I have more bandwidth for this :blush: thanks for the reviews.

**Pull request closed without merging**